opentelemetry-exporter-cloud-monitoring/src/opentelemetry/exporter/cloud_monitoring/__init__.py

```diff
@@ -237,6 +246,11 @@ def _set_start_end_times(self, point, record, metric_descriptor):
                     self._last_updated[updated_key] + int(1e6),
                     NANOS_PER_SECOND,
                 )
+        else:
+            (
+                point.interval.start_time.seconds,
+                point.interval.start_time.nanos,
+            ) = (seconds, nanos)
```

Comment:
nit a little cleaner
```suggestion
            point.interval.start_time.seconds = seconds
            point.interval.start_time.nanos = nanos
```